Newsletter Summer '23 (#699)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2023-07-21-newsletter.md

---
layout: post
title: Summer 2023 Newsletter
editor: Robert Chisholm
slug: 2023-07-21-newsletter
date: 2023-07-21T00:00:00Z+0100
tags:
category:
link:
description:
type: text
---

## The University of Sheffield Research Software Engineering Community Newsletter Summer 2023

Welcome to the summer newsletter for the research software community at The University of Sheffield, featuring news, opportunities, events and training for you.

### News, Web & Blogs

#### Web and Publications

> * Our own [David Wilby](/contact/david-wilby) has published the results of his survey of researchers using MATLAB on [his blog](https://reproduciblematlab.github.io/blog/posts/2023-05-02-survey/).

Reviewer: I wonder if some of these can be grouped into some sensible subsets.

```
#### MATLAB

* Our own [David Wilby](/contact/david-wilby) has published the results of his survey of researchers using MATLAB on [his blog](https://reproduciblematlab.github.io/blog/posts/2023-05-02-survey/).
* [MATLAB toolbox development best practices guide](https://github.com/mathworks/toolboxdesign).

#### Open Research

* [How Can Open Source Program Offices (OSPOs) Support Research Software?](https://www.researchsoft.org/blog/2023-06/)
* The Library's open research team have put together a thorough [guide to understanding the FAIR principles of publishing research outputs and data](https://sites.google.com/sheffield.ac.uk/fair-guidance/home).
* [Open Science: A Practical Guide for Early-Career Researchers](https://zenodo.org/record/7716153#.ZFzsHuxByLp) - L. Brinkman et al.
* [How Can Open Source Program Offices (OSPOs) Support Research Software?](https://www.researchsoft.org/blog/2023-06/)
* [Open source is a hard requirement for reproducibility](https://www.r-bloggers.com/2022/11/open-source-is-a-hard-requirement-for-reproducibility/)
* [FAIR for Jupyter Notebooks: A Practical Guide](https://ardc.edu.au/resource/fair-for-jupyter-notebooks-a-practical-guide/)
* [Materials from CERN/NASAs Summit on Accelerating the Adoption of Open Science](https://zenodo.org/communities/cern-nasa-open-science-summit-july-2023/?page=1&size=20)
* [ROSiE General Guidelines on Responsible Open Science are now available!](https://rosie-project.eu/two-important-milestones-of-the-rosie-project-are-now-available/)
* [ResearchEquals](https://www.researchequals.com/) is a platform to support the open publishing of research at all stages.

#### Blogs
* Lost and Frustrated but Persistent: ([Part 1](https://software.ac.uk/blog/2023-07-03-lost-and-frustrated-persistent-part-1-personal-narratives-about-usability), [Part 2](https://software.ac.uk/blog/2023-07-04-lost-and-frustrated-persistent-part-2-personal-narratives-about-usability))
* [Research Software Security Snippets #2](https://software.ac.uk/blog/2023-07-17-research-software-security-snippets-2) - Alex Coleman

#### Publications

* [Better Research Software Tools to Elevate the Rate of Scientific Discovery -- or why we need to invest in research software engineering](https://arxiv.org/abs/2307.03934) - J. Deschamps, D. D. Nogare, F. Jug
* [The Future of Academic Publishing](https://www.nature.com/articles/s41562-023-01637-2) - A. Ahmed et al.
```
Rest could remain under `#### Web and Publications` unless any other groupings can be spotted.

Author: Seems like a good idea (I originally misread titles and had it split into web + pubs). Will probably get back on this tomorrow.

### Digital Research Practice Support Community

The DRPS community is a group for people that support researchers in carrying out research in the digital age. Meetings are held monthly, with discussions around events, training and opportunities related to the field.

You can join the google group [here](https://groups.google.com/u/1/a/sheffield.ac.uk/g/digital-research-practice-support-community-group/about) to stay informed.

The next meeting is scheduled for 2pm on Wednesday 9th August.

### LunchBytes

[LunchBytes](https://rse.shef.ac.uk/community/lunch-bytes/) are a monthly series of short talks from the research community on research software, data, and infrastructure.

#### LunchBytes needs YOU!

LunchBytes are organised by and for the research software community at The University of Sheffield. If you'd like to curate a session on a topic or present something, get in touch by emailing [lunchbytes-organisers-group@sheffield.ac.uk](mailto:lunchbytes-organisers-group@sheffield.ac.uk) - Or suggest topics [on the jamboard](https://jamboard.google.com/d/1-51cRf0pwZl8O10CnLeJGAqKcnbww-QGaYjszFK-H38/).

### Support

#### Code Clinics

> Why not come to a [Code Clinic](https://docs.google.com/forms/d/e/1FAIpQLScGXS55qjU0D0Zcz-KHOVcNTahcr3YC3H0OpoKBo3lWXWED5A/viewform)? We're keen to hear from you.

Reviewer: Incredibly minor and I've not noticed this before and just copy and pasted previous text for this section but this might be a minuscule improvement...

```suggestion
Why not come to a [Code Clinic](https://docs.google.com/forms/d/e/1FAIpQLScGXS55qjU0D0Zcz-KHOVcNTahcr3YC3H0OpoKBo3lWXWED5A/viewform)? We're keen to help you.
```

* [Research software engineering accelerates the translation of biomedical research for health](https://www.nature.com/articles/s41591-023-02353-0) - D. Horsfall et al.
* [Quarto-based template for university courses](https://github.com/jonjoncardoso/quarto-template-for-university-courses)
* StackOverflow has published the [results of their 2023 developer survey](https://stackoverflow.blog/2023/06/13/developer-survey-results-are-in/).
* [The binary split between academic and professional services staff is too simple for today's complex research environment.](https://twitter.com/sjh5000/status/1678747143656357889)
* Check out the [Code for Thought Podcast](https://codeforthought.buzzsprout.com/1326658), weekly episodes covering Research, (Open) Science & Engineering.

#### MATLAB

* Our own [David Wilby](/contact/david-wilby) has published the results of his survey of researchers using MATLAB on [his blog](https://reproduciblematlab.github.io/blog/posts/2023-05-02-survey/).
* [MATLAB toolbox development best practices guide](https://github.com/mathworks/toolboxdesign).

> 

Reviewer 2: Just a last minute one to add if there's still time?

```suggestion
* Open and run any public MATLAB code from GitHub in MATLAB Online and run *without a license*. Big news for reproducibility in the world of MATLAB. Details in Mike Croucher's blog: [Open science and reusable research with MATLAB Online and GitHub](https://blogs.mathworks.com/matlab/2023/07/20/open-science-and-reusable-research-with-matlab-online-and-github/)](https://blogs.mathworks.com/matlab/2023/07/20/open-science-and-reusable-research-with-matlab-online-and-github/)

```

title: Summer 2023 Newsletter
editor: Robert Chisholm
slug: 2023-07-21-newsletter
date: 2023-07-21T00:00:00Z+0100
tags:
category:
link:
description:
type: text
---

## The University of Sheffield Research Software Engineering Community Newsletter Summer 2023

Welcome to the summer newsletter for the research software community at The University of Sheffield, featuring news, opportunities, events and training for you.

### News, Web & Blogs

#### Web & Publications

* [University of Sheffield's request for REF 2028 feedback](https://staff.sheffield.ac.uk/news/share-your-views-about-ref-2028)
* Shiny apps can now easily integrate with [hugging face](https://shiny.posit.co/blog/posts/shiny-on-hugging-face/)!
* Fostering an Inclusive Culture: [Call for Participation](https://contributor-experience.org/docs/posts/dei-report/)
* The [NASA JPL Observational Products for End-Users from Remote Sensing Analysis (OPERA)](https://www.jpl.nasa.gov/news/nasa-led-project-tracking-changes-to-water-ecosystems-land-surface) datasets are [starting to be released this year](https://www.jpl.nasa.gov/go/opera/products). These are satellite-based water-surface and topography maps through time.

> * [Research software engineering accelerates the translation of biomedical research for health](https://www.nature.com/articles/s41591-023-02353-0) - D. Horsfall et al.

Reviewer: ```suggestion
```

* [Quarto-based template for university courses](https://github.com/jonjoncardoso/quarto-template-for-university-courses)
* StackOverflow has published the [results of their 2023 developer survey](https://stackoverflow.blog/2023/06/13/developer-survey-results-are-in/).
* [The binary split between academic and professional services staff is too simple for today's complex research environment.](https://twitter.com/sjh5000/status/1678747143656357889)
* Check out the [Code for Thought Podcast](https://codeforthought.buzzsprout.com/1326658), weekly episodes covering Research, (Open) Science & Engineering.

> 

Reviewer: ```
#### RSE Advocacy

* [Research software engineering accelerates the translation of biomedical research for health](https://www.nature.com/articles/s41591-023-02353-0) - D. Horsfall et al.
* [Advocating for Equality of Contribution: The Research Software Engineer (RSE) (Heliophysics Decadal Survery 2024)](https://essopenarchive.org/users/564792/articles/618963-advocating-for-equality-of-contribution-the-research-software-engineer-rse-heliophysics-decadal-survey-2024)
```

Author: Doesn't appear you did a suggested change, so I can't apply it.

I added it to my comment above, so will make the same change locally now and merge.

Reviewer: :facepalm: Sorry got ahead of myself just typing markdown.
